avaframe/com1DFA/DFAtools.py

```diff
@@ -5,6 +5,7 @@
 # Load modules
 import logging
 import numpy as np
+import numbers
```

Comment:
where does numbers come from? Does it need to go to the requirements?

Reply:
should be already installed. I use this to check if a value is a float, int double... or a string or other none numeric values